lib/src/neu/generators/openapi2/spectral.ruleset.yml

```diff
@@ -0,0 +1,21 @@
+# https://stoplight.io/p/docs/gh/stoplightio/spectral/docs/reference/openapi-rules.md
+extends: [[spectral:oas, off]]
+rules:
+  operation-2xx-response: true
+  operation-operationId-unique: true
+  operation-parameters: true
+  path-params: true
+  example-value-or-externalValue: true
+  no-eval-in-markdown: true
+  no-script-tags-in-markdown: true
+  openapi-tags-alphabetical: true
+  operation-operationId-valid-in-url: true
+  path-declarations-must-exist: true
+  path-keys-no-trailing-slash: true
+  path-not-include-query: true
+  oas2-operation-formData-consume-check: true
+  oas2-operation-security-defined: true
+  oas2-valid-example: true
+  oas2-anyOf: true
+  oas2-oneOf: true
+  oas2-schema: true
```

Comment:
oh what is this file used for?

Reply:
This is used by Spectral for linting. The old generator uses Speccy, however we introduced Spectral into the new OpenAPI 3 generator (why? see https://github.com/airtasker/spot/issues/629). Spectral is a general purpose linter that supports programmatic usage (Speccy did not, so was used externally by CI)